src/Prestige.Identity.WebApi/Controllers/IdentityController.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Prestige.Identity.Common.Commands;
+using Prestige.Identity.Infrastructure.Services;
+using System.Threading.Tasks;
+
+namespace Prestige.Identity.WebApi.Controllers
+{
+    [Route("identity")]
```

Comment:
By convention, a RESTful route should start with "api/*controller*/*action*", so you can do the Route attribute like this - `[Route("api/[controller]"]`. The `[controller]` in the string will directly put the current Controller name in the route.